app.py

```diff
@@ -135,6 +135,23 @@ def render_and_submit_path(self, path, fields, first_frame, last_frame, sg_publi
         if version_template:
             version_name = version_template.apply_fields(fields)
 
+        sg_entity_type = self.context.entity["type"]
```

Comment:
entity is not guaranteed to be defined, this should be wrapped in a conditional or try/except

Reply:
Good point. Checking it now.